### PR TITLE
feature: realize JWT token

### PR DESCRIPTION
**Pull request by moyuerrr**

### CODE REVIEW

**Review by RuipuCui — APPROVED** [review]
looks good to me :)

File: src/main/java/com/chen/itproject/util/JwtUtils.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package com.chen.itproject.util;
+
+import io.jsonwebtoken.*;
+import org.springframework.stereotype.Component;
+
```
Comment (Leshangpang) [inline src/main/java/com/chen/itproject/util/JwtUtils.java:5]: Maybe remove the extra line here for better formatting

**Review by Leshangpang — APPROVED** [review]
Please consider writing descriptions of the functions and adding javadoc/comments where appropriate in the future.

File: src/main/java/com/chen/itproject/util/JwtUtils.java (hunk 1 above)
Comment (moyuerrr) [reply src/main/java/com/chen/itproject/util/JwtUtils.java:5]: got it

**Pull request merged**